src/frame.js

```diff
@@ -0,0 +1,27 @@
+class Frame {
+
+  constructor() {
+    this.roll_one = 0;
+    this.roll_two = 0;
+  }
+
+  firstRoll(roll) {
+    this._guardClauses(roll);
+    this.roll_one = roll
+  }
+
+  secondRoll(roll) {
+    this._guardClauses(roll);
+    this.roll_two = roll
+  }
+
+  _guardClauses(roll) {
```

Comment:
nicely designed function, I only thought about validating user input late yesterday and i was too tired by then,